index.html

```diff
@@ -0,0 +1,280 @@
+                            </div>
+                        </div>
+                        <div class="review">
+                            <img src="images/avatar-two.png" alt="avatar" class="review__img" loading="lazy">
+                            <div class="review__content">
+                                <h4 class="review__title title-color">Hanna Wilson</h4>
+                                <a href="#" class="review__link">wordfaang.com</a>
+                                <p class="review__text interactive text-color">Suspendisse ultrices at diam lectus nullam.</p>
+                                <p class="review__text interactive text-color">Nisl, sagittis viverra enim erat tortor ultricies massa turpis. Arcu pulvinar aenean nam laoreet nulla.</p>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+            </main>
+            <footer class="footer">
+                <div class="footer__heading">
+                    <div>
+                        <h2 class="footer__title title title-color">Try for free!</h2>
+                        <p class="footer__text text text-color">Get limited 1 week free try our features!</p>
+                    </div>
+                    <div class="footer__button">
+                        <button class="button text button-color-pink" type="button">Learn more</button>
+                        <button class="button-demo interactive" type="button">
+                            Request Demo
+                            <svg class="button-demo-arrow" width="22" height="16" viewBox="0 0 22 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path d="M15 15L21 8L15 1" stroke="#9C69E2" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                                <path d="M21 8H1" stroke="#9C69E2" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                            </svg>
+                        </button>
+                    </div>
+                </div>
+                <div class="footer__navigation">
+                    <div class="contacts">
+                        <div class="contacts__logo">
+                            <img src="images/logo.svg" alt="logo" loading="lazy">
+                            <p class="contacts__text title-color">DataWarehouse</p>
+                        </div>
+                        <div>
+                            <p class="text title-color">Warehouse Society, 234</p>
+                            <p class="text title-color">Bahagia Ave Street PRBW 29281</p>
+                            <a class="contacts__link title-color" href="mailto:info@warehouse.project">info@warehouse.project</a>
+                            <a class="contacts__link title-color" href="tel:1-232-3434">1-232-3434 (Main)</a>
```

Comment:
"-" should be replaced with "&#8209;" (non-breaking hyphen) in telephone numberhtml-validate(tel-non-breaking)